BuildAndTest.cmd

```diff
@@ -19,18 +19,20 @@ if "%ERRORLEVEL%" NEQ "0" (
 )
 popd
 
-
 pushd .\src\security_utilities_rust_ffi\
 call cargo clean --release
-call cargo build --release
+call cargo build --release --target x86_64-pc-windows-msvc
```

Comment:
>build [](http://example.com/codeflow?start=11&length=5)

you don't test the error code for the first build. it could fail, the second build succeeds and sets errorlevel to 0

Reply:
Good catch. Fixed.